apps/web/pages/login/index.tsx

```diff
@@ -0,0 +1,29 @@
+import Head from 'next/head';
+
+import styled from 'styled-components';
+import { useAuth } from '../../context/AuthContext';
+import { Button, Header } from 'ui';
+import Login from '../../components/Login';
+
+export default function Web() {
+  const { user, logout } = useAuth();
+
+  const LoginPageWrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    justify-content: center;
+  `;
```

Comment:
I would suggest to create a WrapperComponent and add an variant="loginPage" or variant "flexColumn".

Like this:

```jsx
const StyledWrapper = styled.div`
	display: flex;
	${({ variant }) =>
		variant === 'loginPage' &&
		css`
                  flex-direction: column;
                   align-items: flex-start;
                   justify-content: center;
`}
	`

export default StyledWrapper;
```

Maybe there are more Wrapper, that have the same style. Take a look at register page.